### DISCUSSION TITLE
ssh remotely to Pi4 from mobile phone

### DISCUSSION THREAD

**Discussion by Doc-coding** [isontheline/pro.webssh.net#1122] [Q&A]
I am trying to connect remotely from my mobile to my Pi4 running bookworm. I use remote.it to access from any network. It works fine with laptop but on mobile phone I keep getting error dns can not be retrieved from ip address. I would be grateful for guidance

**Comment by isontheline** [comment]
Hello @Doc-coding 👋 

I will try to help you as much as you need.

Please share your verbose logs : 
WebSSH > Settings > Log Reports > ... > Level > Verbose

Have a nice day ☀️

**Reply by Doc-coding** [reply]
Please see pics of the log in verbos![image](https://github.com/isontheline/pro.webssh.net/assets/62648007/edec432d-2339-4c4c-aca2-8b3b8de5e1f8)![image](https://github.com/isontheline/pro.webssh.net/assets/62648007/380a63cd-7fc9-4e37-b9fb-5e38e9240c21)![image](https://github.com/isontheline/pro.webssh.net/assets/62648007/91e09e0a-4606-4e4e-adaf-184b3616eb0a)

**Comment by isontheline** [comment]
You're trying to connect to 127.0.0.1 (localhost), what's your DNS you are trying to use please?

**Comment by Doc-coding** [comment]
Hi. Thank you for the quick response. I will explain what I am trying to
do. I am controlling my Raspberry Pi remotely via a programme called
remote.it which allows me various connections VNC, HTTP, SSH. So far if I
use a laptop or desktop everything works perfectly. But from iPhone 13 Pro,
I can VNC and http connections. But ssh opens the WebSSH and first asks for
password which I leave empty because I have tried every password example of
my remote.it account and email for registration account and it does not
accept anything. I can connect via VNC and open terminal and use ifconfig
to get the DNS of Raspberry pi4 running on bookworm full 64bit but I you
require and other info as well please let me know. Thank you

Dr Asad Hussain

**Reply by isontheline** [reply]
* Please share the "host / ip" you use inside your WebSSH settings.
* You say that you leave password empty, but which authentication credentials do you need to use with remote.it? (Don't provide credentials themselves but the credentials type remote.it need {password / private key})
* Are you able to connect to your raspberry with an alternative SSH client like Termius on the App Store?

**Comment by Doc-coding** [comment]
Hi. The ip/host I use is 127.0.0.1 and in the WebSSH setting I enter the
password of my pi with username.
I tried termius but as remote.it opened WebSSH I tried WebSSH so never
tried with Teemis in detail. When I mentioned the password leaving it blank
is when I connect with remote.it app the ssh option it automatically pops a
window with proxy details and asks for password, I leave that blank as when
I enter anything as I mentioned in my previous email, it said wrong
credentials straight away but when I press connect without password, it
moves forward so that I can enter WebSSH details.

Dr Asad Hussain

**Reply by isontheline** [reply]
I think you can't use 127.0.0.1 as it is your iPhone local IP and not your remote.it or rapsberry pi address

**Comment by Doc-coding** [comment]
Hi. The ip I provided is used by remote. It and with the same ip I can
access ssh on my laptop or desktop. But thank you for your prompt response.

Dr Asad Hussain
